templates/branding/openqa.suse.de/docbox.html.ep

```diff
@@ -4,7 +4,7 @@
 <p><a href="https://wiki.microfocus.net/index.php/OpenQA">» Description of internal setup and administration</a></p>
 <p><a href="irc://irc.suse.de/testing">» IRC channel</a></p>
 <p>»
-	<a href="https://build.suse.de/project/staging_projects/SUSE:SLE-12-SP3:GA">SLE</a> and
+	<a href="https://build.suse.de/project/staging_projects/SUSE:SLE-15:GA">SLE15</a> and
```

Comment:
Please add a space between SLE and the number.

And while you're at it: casp20 is released - casp30 is in staging